firebase.json

```diff
@@ -0,0 +1,7 @@
+{
+  "$schema": "./node_modules/@react-native-firebase/app/firebase-schema.json",
+  "react-native": {
+    "messaging_auto_init_enabled": true,
+    "messaging_ios_auto_register_for_remote_messages": true
```

Comment:
You didn't have this configuration before, right? Why do we need this now?

Reply:
You right, we didn't have this explicit configuration before, but it was the default behavior of RNFB. Now we are enforcing this configuration because the RNFB implementation requires to register the iOS devices on APNS prior to register the device on FCM, and this causes a point of failure for the application to handle, that's why this configuration must be explicit now.